app/views/runs/_title.slim

```diff
@@ -1,50 +1,67 @@
-h3
-  = run.to_s
-- if run.completed?(Run::REAL) && run.completed?(Run::GAME)
-  = render partial: 'runs/timing_switch', locals: {timing: timing}
-h5.mt-3
-  div = format_milliseconds(run.duration_ms(timing))
+h1 = run.to_s
+h5
   - if run.user.try(:name)
-    small
-      span by #{link_to run.user, user_path(run.user), class: 'stealth-link user-link'}
-      - if run.user.silver_patron?
-        .gold-split title='Patron'
-h6
+    span> by
+    span = user_badge(run.user)
+    .badge
+      i.fas.fa-check>
+      span Personal Best
+  = render partial: 'runs/timing_badge', locals: {run: run, timing: timing}
```

Comment:
On some pages this is on its own line above everything else, others it's inline with the username and stuff.

![2018-06-09_23-51-46](https://user-images.githubusercontent.com/7030913/41199063-2dda43c8-6c40-11e8-9a5d-a9212155c7d3.png)

![2018-06-09_23-51-54](https://user-images.githubusercontent.com/7030913/41199065-2ffbc974-6c40-11e8-8509-60e4b75e8e1e.png)

Reply:
In the first image there's no user or Personal Best label, so it's the same line but it's just otherwise empty. This seems correct.

Reply:
Question then: should this page also say something like `by: ???` like it does on the game page or should it just be blank?

Reply:
Yeah perhaps, I can change this.